src/points/selectors.ts

```diff
@@ -10,10 +15,17 @@ export const pointsHistoryStatusSelector = (state: RootState) => {
   return state.points.getHistoryStatus
 }
 
-export const pointsHistorySelector = (state: RootState) => {
+const allPointsHistorySelector = (state: RootState) => {
   return state.points.pointsHistory
 }
 
+export const pointsHistorySelector = createSelector(
+  [allPointsHistorySelector],
+  (pointsHistory): ClaimHistory[] => {
+    return pointsHistory.filter((history) => isClaimActivityId(history.activityId))
```

Comment:
i wonder if we need this here if it can be in the saga already (but there's an open thread about that, i might be missing something)

Reply:
see my other comment!